examples/binaries/calc-hash/over-blocks/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "demo-calc-hash-over-blocks"
+version = "0.1.0"
+authors = ["Gear Technologies"]
+edition = "2018"
+license = "GPL-3.0"
+workspace = "../../../../"
+
+[dependencies]
+gstd = { path = "../../../../gstd", features = ["debug"] }
+
+# encoding
+codec = { package = "parity-scale-codec", version = "3.1.2", default-features = false, features = ["derive"]}
+shared = { path = "../", package = "demo-calc-hash" }
+
+[build-dependencies]
+gear-wasm-builder = { path = "../../../../utils/wasm-builder" }
+
+[features]
+std = []
+default = ["std"]
```

Comment:
Can not go without this `features` declaration if wanna to test these programs inside `pallet/gear/tests.rs`, each of the testing programs have this.